pxr/usdImaging/usdImagingGL/drawModeAdapter.cpp

```diff
@@ -1095,8 +1095,51 @@ UsdImagingGLDrawModeAdapter::_ComputeExtent(UsdPrim const& prim) const
 
     TfTokenVector purposes = { UsdGeomTokens->default_, UsdGeomTokens->proxy,
                                UsdGeomTokens->render };
-    UsdGeomBBoxCache bboxCache(UsdTimeCode::EarliestTime(), purposes, true);
-    return bboxCache.ComputeUntransformedBound(prim).ComputeAlignedBox();
+
+    if (prim.IsLoaded()) {
```

Comment:
I'm leaving this note partly for other reviewers, but can you add an XXX comment here saying that it seems problematic that we are using using EarliestTime() here?  

Not something to change as part of this feature, but it definitely strikes me as a problem for animated models.